### PR TITLE
Fix sanor kyy

### PR DESCRIPTION
**Pull request by Ring-k**
fix code smell

### CODE REVIEW

File: jdbc/src/main/java/org/apache/iotdb/jdbc/Utils.java [hunk 1]
```diff
@@ -39,6 +39,13 @@
  */
 public class Utils {
 
+  /**
+   * Private constructor of Utils Class.
+   */
+  private  Utils(){
+    throw new IllegalAccessError("Utility class");
```
Comment (mdf369) [inline jdbc/src/main/java/org/apache/iotdb/jdbc/Utils.java]: Why throwing exception instead of leaving it empty?

File: mvnw.sh [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 #
+#
```
Comment (mdf369) [inline mvnw.sh]: What's this for?

**Comment by Beyyes** [comment]
What's the meaning of `kky`?

File: jdbc/src/main/java/org/apache/iotdb/jdbc/Utils.java (hunk 1 above)
Comment (Ring-k) [reply jdbc/src/main/java/org/apache/iotdb/jdbc/Utils.java]: This is a utility class, which means it provides methods and attributes directly. No class should be able to construct an instance of it, even in a method of itself.

File: mvnw.sh (hunk 2 above)
Comment (Ring-k) [reply mvnw.sh]: It just changed automatically. Didn't modify it. Just ignore it, pls.

File: mvnw.sh [hunk 3]
```diff
@@ -266,4 +267,4 @@ exec "$JAVACMD" \
   $MAVEN_OPTS \
   -classpath "$MAVEN_PROJECTBASEDIR/.mvn/wrapper/maven-wrapper.jar" \
   "-Dmaven.home=${M2_HOME}" "-Dmaven.multiModuleProjectDirectory=${MAVEN_PROJECTBASEDIR}" \
-  ${WRAPPER_LAUNCHER} $MAVEN_CONFIG "$@"
+  ${WRAPPER_LAUNCHER} $MAVEN_CONFIG "$@"
```
[the remark this answers is not in the document]
Comment (Ring-k) [reply mvnw.sh]: Yes, this is unexpected.

File: Readme.md [hunk 4]
```diff
@@ -21,7 +21,7 @@
 
 # IoTDB
 [![Build Status](https://www.travis-ci.org/apache/incubator-iotdb.svg?branch=master)](https://www.travis-ci.org/apache/incubator-iotdb)
-[![codecov](https://codecov.io/gh/thulab/iotdb/branch/master/graph/badge.svg?token=tBhPhPC9EQ)](https://codecov.io/gh/thulab/iotdb)
+[![codecov](https://codecov.io/gh/thulab/incubator-iotdb/branch/master/graph/badge.svg)](https://codecov.io/gh/thulab/incubator-iotdb)
```
[the remark this answers is not in the document]
Comment (Ring-k) [reply Readme.md]: I didn't mean to modify it.

File: jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBDatabaseMetadata.java [hunk 5]
```diff
@@ -122,107 +127,107 @@ private ResultSet getColumnsFunc(String catalog, String schemaPattern)
   @Override
   public boolean isWrapperFor(Class<?> arg0) throws SQLException {
     // TODO Auto-generated method stub
-    throw new SQLException("Method not supported");
+    throw new SQLException(Method_Not_Supported_String);
```
[the remark this answers is not in the document]
Comment (Ring-k) [reply jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBDatabaseMetadata.java]: Oh, sorry. I will install sonarlint and modify it.

File: jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBDatabaseMetadata.java [hunk 6]
```diff
@@ -1193,6 +1199,7 @@ public String toString() {
     try {
       return getMetadataInJsonFunc();
     } catch (IoTDBSQLException e) {
+      LOGGER.info("Failed to fetch metadata in json because: ", e);
```
Comment (jixuan1989) [inline jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBDatabaseMetadata.java]: LOGGER.error is better for Line 1202 1211 and 1220

Comment (Ring-k) [reply jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBDatabaseMetadata.java]: Thanks. I'll fix it.

**Pull request merged**